src/Examples/Flowsheets/ngcc/soec.py

```diff
@@ -1,1233 +1,1226 @@
+        ]
+        for pp in generic_prop_packages:
+            pp.set_default_scaling("enth_mol_phase", 1e-3)
+            pp.set_default_scaling("pressure", 1e-5)
+            pp.set_default_scaling("temperature", 1e-2)
+            pp.set_default_scaling("flow_mol", 1e-3)
+
+        _mf_scale = {"Ar": 100, "O2": 10, "N2": 10, "H2": 10, "H2O": 100, "CO2": 1000}
+        for comp, s in _mf_scale.items():
+            self.o2_side_prop_params.set_default_scaling(
+                "mole_frac_comp", s, index=comp
+            )
+            self.o2_side_prop_params.set_default_scaling(
+                "mole_frac_phase_comp", s, index=comp
+            )
+
+        self.h2_side_prop_params.set_default_scaling("mole_frac_comp", 10)
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_phase_comp", 10)
+
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_comp", 1)
+        self.h2_pure_prop_params.set_default_scaling("mole_frac_phase_comp", 1)
+
+    def _define_cell_params(self):
+        self.soec_module.number_cells.fix(1e6)#1.8*4.4e5)
+
+        self.soec_module.solid_oxide_cell.fuel_channel.length_x.fix(0.002)
+        self.soec_module.solid_oxide_cell.length_y.fix(0.2345)
+        self.soec_module.solid_oxide_cell.length_z.fix(0.2345)
+        self.soec_module.solid_oxide_cell.fuel_channel.heat_transfer_coefficient.fix(100)
+
+        self.soec_module.solid_oxide_cell.oxygen_channel.length_x.fix(0.002)
+        self.soec_module.solid_oxide_cell.oxygen_channel.heat_transfer_coefficient.fix(100)
+
+        self.soec_module.solid_oxide_cell.fuel_electrode.length_x.fix(1e-3)
+        self.soec_module.solid_oxide_cell.fuel_electrode.porosity.fix(0.326)
+        self.soec_module.solid_oxide_cell.fuel_electrode.tortuosity.fix(3)  # Revisit
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_heat_capacity.fix(595)
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_density.fix(7740.0)
+        self.soec_module.solid_oxide_cell.fuel_electrode.solid_thermal_conductivity.fix(6.23)
+        self.soec_module.solid_oxide_cell.fuel_electrode.resistivity_log_preexponential_factor.fix(
+            pyo.log(2.5e-5)
+        )
+        self.soec_module.solid_oxide_cell.fuel_electrode.resistivity_thermal_exponent_dividend.fix(0)
+
+        self.soec_module.solid_oxide_cell.oxygen_electrode.length_x.fix(40e-6)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.porosity.fix(0.30717)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.tortuosity.fix(3.0)  # Revisit
+        # Heat capacity and heat transfer coefficients of oxygen electrode aren't well known but probably don't matter
+        # because the electrode is extremely thin
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_heat_capacity.fix(142.3)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_density.fix(3030)
+        self.soec_module.solid_oxide_cell.oxygen_electrode.solid_thermal_conductivity.fix(5.84)

```

Comment:
Should have:
``` 
soec.oxygen_electrode.solid_density.fix(5300)
soec.oxygen_electrode.solid_thermal_conductivity.fix(2.0)
```
but those numbers unlikely to impact anything because we're not doing dynamics and the electrode is so thin.